src/main/java/com/labsynch/labseer/service/StandardizationServiceImpl.java

```diff
@@ -764,23 +749,27 @@ public int restandardizeParentStructuresBatch(List<Long> pIdGroup) throws CmpdRe
 
 			parent.setMolFormula(cmpdRegMolecule.getFormula());
 
-			parent.merge();
```

Comment:
You don't actually need to call merge, this is all handled by hibernate and it slows things down.